PromoExchange/batman
# Key not keys - UPS Ground only

Ensure we use the correct passed in shipping option.

### Test plan
1. Go to VFA store
1. Select Cups-On-The-Go -16 oz. Stadium Cup 
1. Ensure price quoted on new page is the same as the purchase page
1. Go to other stores, especially anchorfree and ensure price is the same on product page as purchase screen.
